app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter!
+    author.update(posts_counter: author.posts.count)
```

Comment:
- [ ] Right now, your post counter isn't get updated. It remains zero even if the user has a post.
![image](https://github.com/EvansSnave/Blog_app_rails/assets/91486791/c7b379b4-c602-4c81-874f-f1bf1e87d143)
- [ ] Please update the view of your post for a user as it's only showing the title, not body and counters. Kindly implement this wireframe: 
![image](https://github.com/EvansSnave/Blog_app_rails/assets/91486791/0175ed3b-b0cd-451b-9c01-ca5505ccfa7a)